src/ai/backend/test/cli_integration/user/test_vfolder.py

```diff
@@ -71,6 +73,123 @@ def test_rename_vfolder(run_user: ClientRunnerFunc):
     assert bool(test_folder3), "Test folder 3 doesn't exist!"
 
 
+def test_upload_file(run_user: ClientRunnerFunc, make_txt_file: TextIOWrapper):
+    """
+    Test for uploading a file to the vfolder.
+    !! Make sure you execute this test after test_create_vfolder !!
+    Otherwise, it will raise an error.
+    """
+
+    VFOLDER_NAME = "test_folder2"
+    FILE_NAME = make_txt_file.name
+
+    # Upload the file to vfolder
+    with closing(run_user(["vfolder", "upload", VFOLDER_NAME, FILE_NAME])) as p:
+        p.expect(EOF)
+        assert "Done." in p.before.decode(), "File upload failed."
```

Comment:
It is unsafe to check raw messages from the response. I suggest to use `--output=json` option and check `ok` field from the response. Please refer [this](https://github.com/lablup/backend.ai/blob/9df3058fd9981e54737a5905f6e12b83e133c35c/src/ai/backend/test/cli_integration/admin/test_keypair.py#L18-L40) as an example.

Reply:
I wanted to implement in this way, but in vfolder, even if I use the `--output=json` option, the output is not displayed in json. Is there a way to output in json format?

Reply:
@SangwonYoon Sorry, you may leave this as-is now. OutputHandler and `upload()` are not ready for this yet and need to be improved in further PRs. Thank you!